dspace-api/src/main/java/org/dspace/statistics/SolrLoggerServiceImpl.java

```diff
@@ -1691,6 +1691,14 @@ protected synchronized void initSolrYearCores() {
                 statisticYearCores
                     .add(baseSolrUrl.replace("http://", "").replace("https://", "") + statCoreName);
             }
+            var baseCore = ((HttpSolrClient) solr)
+                    .getBaseURL()
+                    .replace("http://", "")
+                    .replace("https://", "");
+            if (!statisticYearCores.contains(baseCore)) {
+                //Also add the core containing the current year, if it hasn't been added already
+                statisticYearCores.add(baseCore);
```

Comment:
@hutattedonmyarm : I'm having a hard time understanding this code change.  It appears (to me) that this `baseCore` will now be added twice.  It's added here on line 1700... but then the same logic appears again on lines 1702-1706 just below this?

I don't understand the benefit of adding it two times... or maybe this code is in the wrong place?

Reply:
You're correct, the lines below need to be removed. I just copied the change from out internal patch and overlooked that part. I'll update the PR in a moment.

I *think* you can test it without previous shards, but you need them enabled. If you enable shards, the base core should be added twice (before the PR) and only once (after the PR), even if no shards from previous years exist